.github/workflows/analysis.yml

```
@@ -0,0 +1,24 @@
+name: "Static Analysis"
+on: ["push", "pull_request"]
+jobs:
+  Static-Check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      - name: Static Code Analysis
+        uses: dominikh/staticcheck-action@v1.1.0
```

We can use the latest minor
```suggestion
        uses: dominikh/staticcheck-action@v1
```